src/assets/scss/blog.scss

```
@@ -141,6 +141,11 @@ ul.tag-list {
     border-block-start: 1px solid var(--divider-color);
 }
 
+// https: //github.com/eslint/eslint.org/issues/257
+// https: //bugzilla.mozilla.org/show_bug.cgi?id=1778180
```

Are these space intentional?

```suggestion
// https://github.com/eslint/eslint.org/issues/257
// https://bugzilla.mozilla.org/show_bug.cgi?id=1778180
```

---

I think that is VS Code trying to format things. Not intentional.